kubernetes/namespaces/monitoring/alerts/alerts.d/django.yaml

```diff
@@ -0,0 +1,20 @@
+groups:
+  - name: django
+    rules:
+      - alert: DjangoErrors
+        expr: increase(django_http_responses_total_by_status_total{status=~"5.."}[5m]) > 0
+        for: 5m
+        labels:
+          severity: page
+        annotations:
+          summary: "Django is experiencing 5xx errors"
+          description: "Django is experiencing 5xx errors on {{ $labels.namespace }}/{{ $labels.job }}"
+
+      - alert: DjangoLatencyHigh
+        expr: histogram_quantile(0.95, rate(django_http_requests_latency_seconds_by_view_method_bucket{view!="api:github-artifacts"}[5m])) > 1.0
```

Comment:
I have a feeling that this will blow up in some cases on endpoint that receive 1 request every full moon and then that request is slow for some reason. Like, I think that we should maybe change this in a way that it will only do the monitoring on endpoints that receive a lot of requests. But I think for now this is fine.

Reply:
Probably yes, someone smarter than me could probably use `django_http_requests_total_by_view_transport_method_total` and validate there have been more than just one-shot requests to a view.